src/main/webapp/app/assessment/assessment-header/assessment-header.component.html

```diff
@@ -181,12 +193,16 @@ <h3 class="top-container flex-wrap flex-lg-nowrap">
                 <fa-icon *ngIf="submitBusy" [icon]="faSpinner" [spin]="true"></fa-icon>
                 <span jhiTranslate="artemisApp.assessment.button.useAsExampleSubmission">Use as Example Submission</span>
             </button>
+            <ng-template #nextSubmissionShortcut
+                >{{ 'artemisApp.assessment.button.control' | artemisTranslate }} + Shift + <fa-icon [icon]="faSquareCaretRight"></fa-icon
+            ></ng-template>
```

Comment:
```suggestion
            <ng-template #nextSubmissionShortcut>
                {{ 'artemisApp.assessment.button.control' | artemisTranslate }} + Shift + <fa-icon [icon]="faSquareCaretRight"></fa-icon>
            </ng-template>
```
Looks like the automatic formatting placed the angles at the wrong places again.